src/tilemaker/README.md

```diff
@@ -0,0 +1,49 @@
+# Tilemaker
+
+This container generates raster tiles using the results of an analysis job. It's run along with
+analysis jobs and uses the shapefiles exported by the analysis.
+
+The easiest way to run it locally is to load the `AnalysisJob` object you're interested in and
+call its `generate_tiles` method.  That won't run the tilemaker command but will print the right
+command to run to the logs, so you can copy it from there and run it.
+
+
+## Adding and Editing Styles
+
+The tiles are defined using Mapnik XML style files in the `styles` directory.
+Since the style XML files aren't templated, the path of the shapefile that's used as the
+data source is hard-coded to a file path in /data/.  The `run_tilemaker.sh` entrypoint script
+downloads the required shapefiles to the expected paths.
+
+For ease of editing and previewing, the styles are defined and maintained in a
+Mapbox Studio Classic project from which the XML style definitions are extracted by hand.
+
+Changes to the working files should be committed so that they're kept in sync with the actual
+tile styles being applied.
+
+### To open and edit the styles
+
+- Download the latest release of Mapbox Studio Classic from https://mapbox.s3.amazonaws.com/mapbox-studio/index.html (the [Mapbox download page](https://www.mapbox.com/mapbox-studio-classic/) links to an older, seemingly-broken version) and install it (for Linux,
```

Comment:
The latest version of Mapbox Studio Classic required that I have a mapbox login to use it, which we don't ever state in the readme. Might be worth mentioning as a prerequisite somewhere.